.github/workflows/cleanup_firebase_on_pr_close.yml

```diff
@@ -0,0 +1,35 @@
+name: Cleanup Firebase Release on PR Close
```

Comment:
In case CI pipeline on a PR failed and there was no deployment on firebase, would this script still work as expected i.e. do nothing?